src/runtime/auto-prompt-manager.js

```diff
@@ -184,41 +191,45 @@ export class AutoPromptManager {
       clientConfig,
       entitlements,
       params.autoPromptType
-    ).then((shouldShowAutoPrompt) => {
-      const potentialActionPromptType = this.getAudienceActionPromptType_({
-        article,
-        autoPromptType: params.autoPromptType,
-        dismissedPrompts,
-        shouldShowAutoPrompt,
-      });
-      const promptFn = potentialActionPromptType
-        ? this.audienceActionPrompt_({
-            action: potentialActionPromptType,
+    )
+      .then((shouldShowAutoPrompt) => {
```

Comment:
This could be simplified with an arrow function expression (https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Functions/Arrow_functions)
```suggestion
      .then((shouldShowAutoPrompt) => Promise.all([
```